src/components/quickLaunch/QuickLaunch.js

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ * @author sriram
+ *
+ */
+import React from "react";
+import Chip from "@material-ui/core/Chip";
+import { withStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import PublicIcon from "@material-ui/icons/Public";
+import LockIcon from "@material-ui/icons/Lock";
+
+const styles1 = (theme) => ({
+    chip: {
+        margin: theme.spacing.unit,
+    },
+});
+
+function QuickLaunch(props) {
+    const {
+        label,
+        handleClick,
+        handleDelete,
+        isDefault,
+        isPublic,
+        classes,
+    } = props;
+    const color = isDefault || isPublic ? "primary" : "";
+    const icon = isPublic ? <PublicIcon /> : <LockIcon />;
+    if (color) {
+        return (
+            <Chip
+                label={label}
+                icon={icon}
+                onDelete={handleDelete}
+                onClick={handleClick}
+                color={color}
+                className={classes.chip}
+                variant="outlined"
+            />
+        );
+    } else {
+        return (
+            <Chip
+                label={label}
+                icon={icon}
+                onDelete={handleDelete}
+                onClick={handleClick}
```

Comment:
Instead of using if/else, does it not work with `Chip` to have something like:
```
            <Chip
                label={label}
                icon={icon}
                onDelete={handleDelete}
                onClick={handleClick}
                color={isDefault || isPublic ? "primary" : null}
                className={classes.chip}
                variant="outlined"
            />
```

Reply:
I tried that. If you supply `null` to color, it start spewing warning about `color` being null.

Reply:
Have you tried `color={color || ""}`? Or maybe even `color={color || undefined}` (not sure about this one 🤔 )?

Reply:
`const color = isDefault || isPublic ? "primary" : "default";` worked!